deploy/scripts/src/authentication/test.ts

```diff
@@ -567,25 +534,13 @@ export function signIn(): void {
         { isStatusCode302 }
       )
 
-      //02_OIDCCall
-      res = timeGroup(groups[15].split('::')[1], () => http.get(res.headers.Location, { redirects: 0 }), {
-        isStatusCode302
-      })
-
-      // 03_RPStub
-      res = timeGroup(groups[16].split('::')[1], () => http.get(res.headers.Location), {
+      // 02_OrchStub
+      res = timeGroup(groups[12].split('::')[1], () => http.get(res.headers.Location), {
         isStatusCode200,
         ...pageContentCheck(userData.email.toLowerCase())
       })
     })
   }
 
-  // 25% of users logout
-  if (Math.random() <= 0.25) {
-    sleep(1)
-
-    // B02_SignIn_07_Logout
-    logout(groups.slice(17, 21))
```

Comment:
Can remove the `logout()` function too if it is no longer being used

Reply:
Removed